config/initializers/env.rb

```diff
@@ -14,7 +14,11 @@
                                 else
                                   ''
                                 end
-  europeana[:entities_api_key] = ENV['EUROPEANA_ENTITIES_API_KEY'] || Europeana::API.key
+
+  europeana[:entities] = OpenStruct.new(
+    api_key: ENV['EUROPEANA_ENTITIES_API_KEY'] || Europeana::API.key,
+    api_url: ENV['EUROPEANA_ENTITIES_API_URL'] || Europeana::API.url
```

Comment:
Good point. I think the better solution is just not to set them if the env vars are not present.